swarm/pss/handshake_test.go

```diff
@@ -30,35 +28,40 @@ func testHandshake(t *testing.T) {
 
 	// set up two nodes directly connected
 	// (we are not testing pss routing here)
-	topic := BytesToTopic([]byte("foo:42"))
-
 	clients, err := setupNetwork(2)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	var loaddr []byte
+	var topic string
+	err = clients[0].Call(&topic, "pss_stringToTopic", "foo:42")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// "0x" = 2 bytes + addrsize address bytes which in hex is 2x length
```

Comment:
This comment seems to be misplaced? I'm not really sure what it is referring to.